kernel/src/process.rs

```diff
@@ -1579,27 +1588,28 @@ impl<C: 'static + Chip> Process<'_, C> {
         let process_name = tbf_header.get_package_name();
 
         // If this isn't an app (i.e. it is padding) or it is an app but it
-        // isn't enabled, then we can skip it but increment past its flash.
+        // isn't enabled, then we can skip it and do not create a `Process`
+        // object.
         if !tbf_header.is_app() || !tbf_header.enabled() {
             if config::CONFIG.debug_load_processes {
                 if !tbf_header.is_app() {
                     debug!(
-                        "[!] flash=[{:#010X}:{:#010X}] process={:?} - process isn't an app",
+                        "Padding in flash=[{:#010X}:{:#010X}]",
                         app_flash.as_ptr() as usize,
-                        app_flash.as_ptr() as usize + app_flash.len(),
-                        process_name
+                        app_flash.as_ptr() as usize + app_flash.len()
```

Comment:
Here for example the flash range doesn't include the end (which I find clearer).